MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -940,6 +941,129 @@ private void OnSetAnimationState(SetAnimationState payload)
                 .SetAnimationState(payload.AnimationName, payload.State.Time, payload.State.Speed, payload.State.Enabled);
         }
 
+
+
+
+        private void ApplySoundStateOptions(AudioSource soundInstance, SoundStateOptions options)
+        {
+            if (options != null)
+            {
+                float test = soundInstance.clip.length;
+                if (options.Volume != null)
+                {
+                    soundInstance.volume = options.Volume.Value;
+                }
+                if (options.Pitch != null)
+                {
+                    //convert from halftone offset (-12/0/12/24/36) to pitch multiplier (0.5/1/2/4/8).
+                    soundInstance.pitch = Mathf.Pow(2.0f, (options.Pitch.Value / 12.0f));
+                }
+                if (options.Looping != null)
+                {
+                    soundInstance.loop = options.Looping.Value;
+                }
+            }
+        }
+
+
+        public static Dictionary<Guid, AudioSource> _soundInstances = new Dictionary<Guid, AudioSource>();
```

Comment:
Underscore naming is for private fields.  Not public static.  Use s_ or no prefix.

Reply:
made it private.

---
In reply to: [260556109](https://github.com/Microsoft/mixed-reality-extension-unity/pull/35#discussion_r260556109) [](ancestors = 260556109)